### PR TITLE
Adds "linked_to_centcomm" variable to maps.

### PR DESCRIPTION
**Pull request by ghost**
- Obviously, by default, this will be one.
- Adds new event where the link with central command is broken for 5 minutes
- Admins can make or break the link with Central command Indefinitely with the secrets panel
- You cannot do the following while the link is broken: Call the shuttle / Recall the shuttle / Fax Central command / Recieve messages from Central command / Send messages to Central Command / Call or Send the Cargo Shuttle
- An admin may send a message to the crew even when the link is broken, but he will be prompted first asking if he is sure.
- Blob alerts ignore whether or not the link with central command is broken

### CODE REVIEW

**Comment by ghost** [comment]
This will be useful for #6789

**Comment by ghost** [comment]
todo:
xpcybic> celt: request for muh immulsions
DrCelt> what's up
xpcybic> if an admin force-sends a centcomm report while the link is broken, garble the centcomm name header a little
DrCelt> In what way?
xpcybic> like "G%%%c\ic P[ot$$to?at< &p@@@e" for "Galactic Protectorate Update"
DrCelt> err
DrCelt> I think I can do that
xpcybic> i mean
DrCelt> I'll steal some stuff from saycode
xpcybic> idk but i think there's something that makes silicons do that when they're emp'ed?

**Comment by 9600bauds** [comment]
He subtly called you a potato
He is making fun of your land

**Comment by ghost** [comment]
new commit:
- Adds more static variety to jukebox
- Adds static to command report when not linked to centcomm
- Gibberishes command report when not linked to centcomm
- Admins can send ungibberished reports, though the title and galactic name will still be gibberish

**Comment by ghost** [comment]
that newest commit hasn't been tested because I can't clear my rsc for some reason

**Comment by ghost** [comment]
or maybe because I forgot to add .ogg to the end whoops

**Comment by ghost** [comment]
todo:
- replace all manual playing of sounds after command alerts and integrate it into the command alert proc
- make static play instead of sounds when not connected with central command
- make default sound played the command report thing

that'll cut down on unnecessary lines and also mean that people in the lobby do not hear the noise of the command report.

**Comment by ihadtoregisterforthis** [comment]
Do you sleep?

**Comment by ghost** [comment]
Coding makes me lose track of time.

**Comment by ghost** [comment]
> can't recall for 5 minutes

Just don't have it give any warning in advance or it will end rounds.

**Comment by ghost** [comment]
It's extremely rare and there's no forewarning, @termedclepe

**Comment by unid15** [comment]
Why not a z-level var instead? Makes more sense

**Comment by ghost** [comment]
@unid15 Not really, considering announcements are global and central command is hardly going to lose connection with the station but keep it with the derelict.

**Comment by PJB3005** [comment]
Same opinion as unid here.

**Comment by ghost** [comment]
done:
-    replace all manual playing of sounds after command alerts and integrate it into the command alert proc
-   make static play instead of sounds when not connected with central command
-   make default sound played the command report thing

**Comment by Imagebard** [comment]
>and there's no forewarning

But why

**Comment by ghost** [comment]
Because it lasts for five minutes nerd, and the shuttle can't be recalled during that time, so add two and two together.

There's an announcement when you've lost connection though

**Comment by Imagebard** [comment]
That's exactly why I'm worried. This has potential to make the dumbest round ends ever.

**Comment by ghost** [comment]
It's extremely rare nerd

**Comment by clusterfack** [comment]
New idea for an event: .1% chance for the station to just automatically have mr clean be summoned and end the round out of nowhere

/sarcasm

File: code/WorkInProgress/kilakk/fax.dm [hunk 1]
```diff
@@ -117,9 +117,10 @@ var/list/alldepartments = list("Central Command")
 /obj/machinery/faxmachine/Topic(href, href_list)
 	if(href_list["send"])
 		if(tofax)
-
-			log_game("([usr]/([usr.ckey]) sent a fax titled [tofax] to [dpt] - contents: [tofax.info]")
 			if((dpt == "Central Command") | (dpt == "Nanotrasen HR"))
+				if(!map.linked_to_centcomm)
+					to_chat(usr, "<span class='danger'>\The [src] displays a 404 error: Central Command not found</span>")
```
Comment (PJB3005) [inline code/WorkInProgress/kilakk/fax.dm:122]: Really?

Comment (ghost) [inline code/WorkInProgress/kilakk/fax.dm:122]: whaaaaaaaaaat

Comment (kilkun) [inline code/WorkInProgress/kilakk/fax.dm:122]: what an awful meme. come on celt apply yourself.

Comment (ghost) [inline code/WorkInProgress/kilakk/fax.dm:122]: ;__;

**Comment by Duny-** [comment]
bluethumbsup

**Comment by Iamgoofball** [comment]
instead of ending after 5 minutes:

make the 404: Central Command Not Found error tell you to reboot the system

this requires depowering and repowering all communications consoles and cargo consoles via either de/re-construction/flipping the breaker switch on the APC off and on again

any rebooted comms/cargo consoles will show the location of the remaining consoles needing a reboot with area and coordinates

makes this last a bit longer/be more involved other than "lol wait 5 minutes for shuttle/cargo console to work again", and allows traitors to build cargo/comms consoles all over the place to extend the "i can break shit" time they're given by a CC Link break

**Comment by ghost** [comment]
Still ready to be merged honestly

**Comment by PJB3005** [comment]
Conflicts nerd.

**Comment by ghost** [comment]
resolved

**Comment by clusterfack** [comment]
:^) 
Conflicts

**Comment by ghost** [comment]
AAAAAA

**Pull request merged**